__tests__/pages/profile/__snapshots__/username.test.js.snap

```diff
@@ -914,13 +914,13 @@ exports[`user profile test Should render loading spinner if data is not ready 1`
             <div
               class="text-uppercase bg-primary rounded-circle text-light user_info_image"
             >
-              fu
+              A 
```

Comment:
There shouldn't be any new snapshots for this test. That's why your checks [failed](https://github.com/garageScript/c0d3-app/pull/520/checks#step:4:30). 
I'm not sure why it's happening but try changing `expect(container).toMatchSnapshot()` to `await waitFor(()=>expect(container).toMatchSnapshot())` in line 60. Maybe jest didn't wait long enough before tearing down everything.